README.md

```diff
@@ -1,70 +1,53 @@
-| [AEPEdgeIdentity Extension](https://github.com/adobe/aepsdk-edgeidentity-android)        | The AEPEdgeIdentity extension enables handling of user identity data from a mobile app when using AEP SDK and the AEP Edge Network extension. |
-| [AEP SDK Sample App for iOS](https://github.com/adobe/aepsdk-sample-app-ios)         | Contains iOS sample apps for the AEP SDK. Apps are provided for both Objective-C and Swift implementations. |
-| [AEP SDK Sample App for Android](https://github.com/adobe/aepsdk-sample-app-android) | Contains Android sample app for the AEP SDK.                 |
+| Project                                                                                            | Description                                                  |
+| -------------------------------------------------------------------------------------------------- | ------------------------------------------------------------ |
+| [Core extensions](https://github.com/adobe/aepsdk-core-android)                                    | The Mobile Core represents the foundation of the Experience Platform Mobile SDK. |
+| [Consent for Edge Network extension](https://github.com/adobe/aepsdk-edgeconsent-android)          | The Consent for Edge Network extension enables consent preferences collection from your mobile app when using the Experience Platform Mobile SDK and the Edge Network extension. |
+| [Lifecycle for Edge Network extension](https://github.com/adobe/aepsdk-core-android)               | The Lifecycle for Edge Network extension helps collect application Lifecycle metrics and any additional context data provided by the application developer when using the Mobile SDK and the Edge Network extension. |
+| [Identity for Edge Network extension](https://github.com/adobe/aepsdk-edgeidentity-android)        | The Identity for Edge Network extension enables handling of user identity data from a mobile app when using the Experience Platform Mobile SDK and the Edge Network extension. |
+| [Adobe Experience Platform Assurance](https://github.com/adobe/aepsdk-assurance-android)           | The Assurance extension enables validation workflows for your Experience Platform Mobile SDK implementation. |
+| [Adobe Experience Platform Android sample app](https://github.com/adobe/aepsdk-sample-app-android) | Contains a fully implemented Android sample app using the Experience Platform Mobile SDK. |
 
 ## Documentation
 
-Additional documentation for usage and SDK architecture can be found under the [Documentation](Documentation) directory.
+Information about Adobe Experience Platform Edge Network's implementation, API usage, and architecture can be found in the [Documentation](Documentation) directory.
```

Comment:
can you include Getting started with the test app in the Documentation/README.md contents table?